Source/WebGPU/WebGPU/RenderPipeline.mm

```diff
@@ -864,7 +863,7 @@ static auto makeBindingLayout(WGPUBindGroupLayoutEntry& newEntry, auto& bindingM
                 return e.binding == entry.webBinding;
             }); existingBindingIndex != notFound) {
                 entries[existingBindingIndex].visibility |= visibility;
-                entries[existingBindingIndex].metalBinding[stage] = entry.binding;
+                std::span(entries[existingBindingIndex].metalBinding)[stage] = entry.binding;
```

Comment:
could also change the definition in WebGPU.h from:
```
uint32_t metalBinding[WGPUShaderStage_Compute / 2 + 1];
```
to
```
std::array<uint32_t, 3> metalBinding;
```
to avoid adding the `std::span` here